convertNuPRISM.C

```diff
@@ -1,7 +1,14 @@
 {
-gROOT->ProcessLine(".L $WCSIMDIR/libWCSimRoot.so");
-gSystem->AddIncludePath(" -I$WCSIMDIR/include/");
-gROOT->ProcessLine(".L nuPRISMconvert.cc+");
-
+  {
+    const char *wcsimdirenv = gSystem->Getenv("WCSIMDIR");
+    if (!(wcsimdirenv && wcsimdirenv[0])) { // make sure it's non-empty
+      wcsimdirenv = "."; // the "default" value
+      std::cout << "Note: WCSIMDIR not set, assuming: " << wcsimdirenv << std::endl;
+      gSystem->Setenv("WCSIMDIR", wcsimdirenv);
+    }
+  }
+  
+  gInterpreter->AddIncludePath("${WCSIMDIR}/include");
+  gSystem->Load("${WCSIMDIR}/libWCSimRoot");
+  gROOT->ProcessLine(".L nuPRISMconvert.cc+");
```

Comment:
Have you checked that these changes to how the library is loaded work with different root versions? Is there a reason for the changes other than preferred style?

Reply:
Yes, this part is mainly a source code "polishing" modification. I decided to do it as I anyhow needed to edit this file to protect it against non-existent "`WCSIMDIR`".